src/components/user-accordion/accordion-item/index.tsx

```diff
@@ -1,114 +1,59 @@
-  const [openLoader, setOpenLoader] = useState(false)
-  const [isOpen, setIsOpen] = useState(false)
-  const descriptionRef = useRef<HTMLDivElement>(null)
-  const [indaxesSum, setIndaxesSum] = useState(0)
-  const [modal, setModal] = useState(false)
-  const [pendingIndex, setPendingIndex] = useState<number | null>(null)
-  const [pendingChecked, setPendingChecked] = useState<boolean | null>(null)
+const AccordionItem = ({ item }: Props) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const descriptionRef = useRef<HTMLDivElement>(null);
+  const [indaxesSum, setIndaxesSum] = useState(0);
 
   useEffect(() => {
-    const count = () => item.checked.reduce((acc, num) => acc + num, 0)
-    setIndaxesSum(count)
-  }, [item.checked])
+    const count = item.checked.reduce((acc, num) => acc + num, 0);
+    setIndaxesSum(count);
+  }, [item.checked]);
 
   const toggleAccordion = () => {
-    setIsOpen(!isOpen)
-  }
-
-  const mutation = useMutation({
-    mutationFn: ({ probaName, probaSubName, probaIndex, value }: any) => {
-      return axios.patch(
-        `${BACKEND_URL}/probas/${currentProbaEmail}`,
-        {
-          probaName,
-          probaSubName,
-          probaIndex,
-          value,
-        },
-        { withCredentials: true }
-      )
-    },
-    onSuccess: (_data, variables) => {
-      item.checked[variables.index] = variables.value
-      const newSum = item.checked.reduce((acc, num) => acc + num, 0)
-      setIndaxesSum(newSum)
-      refetchData()
-    },
-    onError: (error) => {
-      console.error("Error updating checked status:", error)
-    },
-  })
-
-  const handleModalConfirm = () => {
-    if (pendingIndex !== null && pendingChecked !== null) {
-      setOpenLoader(true)
-      mutation.mutate(
-        {
-          probaName: item.probaType,
-          probaSubName: currentStep,
-          probaIndex: pendingIndex,
-          value: pendingChecked ? 1 : 0,
-        },
-        {
-          onSettled: () => {
-            setPendingIndex(null)
-            setPendingChecked(null)
-            setModal(false)
-            setOpenLoader(false)
-          },
-        }
-      )
-    }
-  }
+    setIsOpen(!isOpen);
+  };
 
   return (
     <div className="flex flex-col border-b border-gray-200 w-[95%] mx-auto">
-      <ModalCheckoutButton
-        modal={modal}
-        onLoading={openLoader}
-        setModal={setModal}
-        onConfirm={handleModalConfirm}
-        onCancel={() => setModal(false)}
-      />
       <div
         className="flex justify-between items-center py-4 cursor-pointer"
         onClick={toggleAccordion}
       >
         <span className="text-base font-medium md:font-semibold">
           {item.section} ({indaxesSum} / {item.items.length})
         </span>
-        <button>{isOpen ? <CaretDown className='size-4' /> :<CaretLeft className='size-4' />}</button>
+        <button>
+          {isOpen ? (
```

Comment:
use css rotate instead of conditional rendering, it will provide better readability